docs/manual/constant-value-checker.tex

```diff
@@ -251,6 +251,39 @@ \section{Warnings\label{value-checker-warnings}}
 
 \end{itemize}
 \end{sloppypar}
+
+
+\section{Overflow\label{value-checker-overflow}}
+
+The Constant Value Checker normally takes Java's normal overflow rules into account when
+computing the possible values of expressions.
+The \code{-AignoreRangeOverflow} command-line option
+makes it ignore the possibility of overflow
+for range annotations (\<@IntRange>, for instance), making them more precise.
+You can supply the \code{-AignoreRangeOverflow} command-line option
+when you are confident that no overflow will occur in the program (for
```

Comment:
The programmer doesn't need to know the exact bound in order to write correct annotations.

The reason the Value Checker reasons about overflow is in order to provide a sound estimate.  It might seem on first blush that initialization is independent of nullness, so the Nullness Checker should not reason about initialization; but if it did not, then it would be unsound.  Likewise, the Value Checker would be unsound if it did not reason about the possibility of overflow.